.github/workflows/coverage.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         id: parse-coverage
       - run: echo current coverage is ${{ steps.parse-coverage.outputs.coverage-percent }}%
       - name: Upload lcov build artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
it looks like this [update](https://github.com/actions/upload-artifact/releases/tag/v4.0.0) is a breaking change, since downloading with v3 would no longer work. can you call it out in the PR description? also, have you checked all the repos that use this workflow to make sure this won't affect them? I'd rather not have to make a v2 for this action if we don't need to